packages/core/src/components/TextField/TextField.scss

```diff
@@ -63,6 +63,7 @@ Style guide: components.text-field
 
 /*
 ---
+Inverse theme
```

Comment:
Button is the only other component with a separate code block for inverse theme. Later maybe we should consider combining the inverse examples with the main examples like in `<Dropdown>`?